control-plane-operator/controllers/hostedcontrolplane/kubevirt_assets/files/controller.yaml

```diff
@@ -0,0 +1,125 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: kubevirt-csi-controller
+  namespace: {{NAMESPACE}}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kubevirt-csi-driver
+  template:
+    metadata:
+      labels:
+        app: kubevirt-csi-driver
+    spec:
+      hostNetwork: true
+      serviceAccount: kubevirt-csi
+      priorityClassName: system-cluster-critical
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
```

Comment:
this node selector needs to be removed. We don't care what node the controller runs on in the hosted cluster environment.